cli/azd/cmd/init.go

```diff
@@ -424,14 +464,32 @@ Do not stop until all tasks are complete and fully resolved.
 		},
 	}
 
+	stepsSummaryLabels := []string{
+		"Step 1 (discovery & analysis)",
+		"Step 2 (architecture plan)",
+		"Step 3 (dockerfile generation)",
+		"Step 4 (infrastructure generation)",
+		"Step 5 (azure.yaml generation)",
+		"Step 6 (project validation)",
+	}
```

Comment:
How are these labels different from the `initStep` names above? Are we using both? If we need them both lets consolidate them into that same object.